File: 250324/시험 통과 여부 확인하기 2/verify-test-passed-2.cpp
#include <iostream>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int score[4] = {0,};
    int sum = 0, count = 0;
    double avg = 0;
    int n = 0;
    cin >> n;
    for(int i = 0; i < n; i++) {
        sum = 0;
        avg = 0;
        for(int j = 0; j < 4; j++) {
            cin >> score[j];
            sum += score[j];
        }
        avg = sum / 4.0;
        if(avg >= 60) {
            cout << "pass\n";
            count++;
        }
        else {
            cout << "fail\n";
        }
    }
    cout << count;
    return 0;
}

File: 250324/시험 통과 여부 확인하기 2/verify-test-passed-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int student = sc.nextInt();
        int[] scores = new int[4];
        int sum = 0;
        double avg = 0;
        int count = 0;
        for(int i = 0; i < student; i++) {
            sum = 0;
            avg = 0;
            for(int j = 0; j < 4; j++) {
                scores[j] = sc.nextInt();
                sum += scores[j];
            }
            avg = sum / 4.0;
            if(avg >= 60) {
                System.out.println("pass");
                count++;
            }
            else {
                System.out.println("fail");
            }
        }
        System.out.println(count);
        sc.close();
    }
}